gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -44,5 +44,5 @@ export const GalleryDescriptionTile: FunctionComponent<PropsWithRequiredChildren
 
 export const GalleryExampleTile: FunctionComponent<PropsWithRequiredChildren> =
   function GalleryExampleTile(props: PropsWithRequiredChildren) {
-    return <GalleryTile { ...props } title="Example" />;
+    return <GalleryTile { ...props } title="Example" className="gallery-example"/>;
```

Comment:
Does this class do anything?

Reply:
> Does this class do anything?

Yes, now it controls the code font size: https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/de9176b76764f09ae5e88dc22a9960c63f91627d